doc/nims.rst

```diff
@@ -47,18 +48,50 @@ NimScript is subject to some limitations caused by the implementation of the VM
 * More than one level of `ref` is generally not supported (for example, the type
   `ref ref int`).
 
-* multimethods are not available.
+* Multimethods are not available.
+
+* ``random.randomize()`` requires an ``int64`` explicitly passed as argument, you *must* pass a Seed integer.
+
+* ``unicode`` can be imported, but not ``unidecode``.
 
-Given the above restrictions, at least the following modules are available:
+
+Standard library modules
+========================
+
+At least the following standard library modules are available:
 
 * `macros <macros.html>`_
 * `os <os.html>`_
 * `strutils <strutils.html>`_
 * `math <math.html>`_
 * `distros <distros.html>`_
-
-In addition to the standard Nim syntax (`system <system.html>`_
-module), NimScripts support the procs and templates defined in the
+* `sugar <sugar.html>`_
+* `algorithm <algorithm.html>`_
+* `base64 <base64.html>`_
+* `bitops <bitops.html>`_
+* `chains <chains.html>`_
+* `colors <colors.html>`_
+* `complex <complex.html>`_
+* `htmlgen <htmlgen.html>`_
+* `httpcore <httpcore.html>`_
+* `lenientops <lenientops.html>`_
+* `mersenne <mersenne.html>`_
+* `options <options.html>`_
+* `parseutils <parseutils.html>`_
+* `punycode <punycode.html>`_
+* `random <punycode.html>`_
+* `stats <stats.html>`_
+* `strformat <strformat.html>`_
+* `strmisc <strmisc.html>`_
+* `strscans <strscans.html>`_
+* `unicode <unicode.html>`_
+* `uri <uri.html>`_
+* `std/editdistance <editdistance.html>`_
+* `std/wordwrap <wordwrap.html>`_
+* std/sums
```

Comment:
Do we need to create a documentation page for this one?

Reply:
https://github.com/nim-lang/Nim/issues/11543#issue-458254530

Reply:
[Fixed](https://github.com/nim-lang/Nim/commit/96bed814d2c630bee4eab8bd7a70b0975ada6581), you can add a link.